### PR TITLE
Feat/delete songs

### PR DESCRIPTION
**Pull request by KlaraHajdu**
* delete songs by owner
* delete playlists by owner

### CODE REVIEW

File: src/components/PlaylistPage/PlaylistPage.tsx [hunk 1]
```diff
@@ -23,32 +24,44 @@ const PlaylistPage = () => {
...
     const currentPlaylist = useSelector(selectCurrentPlaylist);
     const currentUser = useSelector(selectCurrentUser);
-    const [addSongActive, setAddSongActive] = useState(false);
-    const [inviteActive, setInviteActive] = useState(false);
-    const [playerActive, setPlayerActive] = useState(false);
-    const [confirmationIsVisible, setConfirmationIsVisible] = useState(false)
+    const [addSongActive, setAddSongActive] = useState<boolean>(false);
+    const [inviteActive, setInviteActive] = useState<boolean>(false);
+    const [playerActive, setPlayerActive] = useState<boolean>(false);
+    const [confirmationIsVisible, setConfirmationIsVisible] = useState<boolean>(false)
```
Comment (bbucsek) [inline src/components/PlaylistPage/PlaylistPage.tsx]: if there is an initial value, the typescipt will figure it out that this is a boolean type

File: src/components/PlaylistPage/PlaylistPage.tsx [hunk 2]
```diff
@@ -23,32 +24,44 @@ const PlaylistPage = () => {
...
 
-    const unFollowOnClick = () => {
+    const deleteOnClick = () => {
         setConfirmationIsVisible(true)
+        setConfirmationText("delete the playlist")
     }
 
     const confirmAction = (isConfirmed: boolean) => {
-        if (isConfirmed){
+        if (isConfirmed && confirmationText.includes("unfollow")){
```
Comment (bbucsek) [inline src/components/PlaylistPage/PlaylistPage.tsx]: this solution seems wrong to me, maybe add a type to the confirmation?

**Review by bbucsek — CHANGES_REQUESTED** [review]

File: src/components/PlaylistPage/PlaylistPage.tsx (hunk 1 above)
Comment (KlaraHajdu) [reply src/components/PlaylistPage/PlaylistPage.tsx]: fixed

File: src/components/PlaylistPage/PlaylistPage.tsx (hunk 2 above)
Comment (KlaraHajdu) [reply src/components/PlaylistPage/PlaylistPage.tsx]: fixed

File: src/components/PlaylistPage/PlaylistPage.tsx [hunk 3]
```diff
@@ -20,14 +20,20 @@ import {
 import Songlist from "../Songlist";
 import Confirmation from "../Confirmation";
 
+enum ActionType {
```
Comment (bbucsek) [inline src/components/PlaylistPage/PlaylistPage.tsx]: cool! small detail, maybe put the ActionType to the types directory, so in the future we can use this enum in other parts of the app?

**Review by bbucsek — CHANGES_REQUESTED** [review]

File: src/components/PlaylistPage/PlaylistPage.tsx (hunk 3 above)
Comment (KlaraHajdu) [reply src/components/PlaylistPage/PlaylistPage.tsx]: fixed

**Review by bbucsek — APPROVED** [review]

**Pull request merged**